packages/web/stores/index.tsx

```diff
@@ -26,6 +26,8 @@ function refetchUserQueries(apiUtils: ReturnType<typeof api.useUtils>) {
   apiUtils.edge.pools.getUserSharePool.invalidate();
   apiUtils.edge.pools.getSharePoolBondDurations.invalidate();
   apiUtils.local.balances.getUserBalances.invalidate();
+  apiUtils.local.bridgeTransfer.getSupportedAssetsBalances.invalidate();
+  apiUtils.edge.assets.getImmersiveBridgeAssets.invalidate();
```

Comment:
Added these to ensure old balance data is not displayed in the d/w flow after a transfer is started / completed. @JoseRFelix LMK if I'm missing any